back-end/GPTapi.js

```diff
@@ -0,0 +1,34 @@
+import { config } from "dotenv";
+import { Configuration, OpenAIApi } from "openai";
+import express from "express";
+import cors from "cors";
+
+config();
+
+const PORT = 8080;
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+
+app.listen(PORT, () => console.log(PORT));
+
+app.post("/chat_input", async (req, res) => {
+  let { initialValue } = req.body;
+
+  const configuration = new Configuration({
+    organization: "org-OCQxyNplVRTgv6vXbnandxBc",
+    apiKey: process.env.GPT_API_KEY,
+  });
+  const openai = new OpenAIApi(configuration);
```

Comment:
You can prob define these configuration stuff outside the `app.post` function so that it is called only once on server startup instead of being called on every function call